docker-compose.yml

```diff
@@ -13,6 +13,11 @@ services:
     volumes: # for data persistence
       - mongodb_vol:/data/db
 
+  redis:
+    image: redis
+    ports:
+      - '6379:6379'
+
```

Comment:
Include container_name and restart like the other containers

    restart: unless-stopped
    container_name: mongodb

![image](https://user-images.githubusercontent.com/77485021/200310754-356730d4-db06-45a5-9300-59cdbc14baea.png)